### ISSUE TITLE
Passing path to markdown folder like CLI argument

### ISSUE THREAD

**Issue by egregors** [freetonik/underblog#10]
It will be much more useful if CLI will expect path to markdown dir like arg, like
```
$ underblog ~/myBlog
...
$ underblog ~/Docs/myCatsBlog
...
$ underblog .
```

**Comment by freetonik** [comment]
I'm a bit hesitant, tbh. You pass md dir, but what about templates dir? Should `underblog` assume template dir is current dir? Or 1 level above md dir? Now we're forced to either introduce another CLI argument for templates or introduce implicit behavior.

I'd love `underblog` to be an opinionated, dumb, barebones tool with virtually zero complexity for the user.

What do you think?

**Comment by egregors** [comment]
I agree. «keep it simple» – the best way to success for such type of programs. But, I believe this kind of customization will be very useful, at least not for MD dir but for an entry point.

For this time we may save the current default behavior, like 
```
$ underblog
```
will expect templates and MD in the current workdir. 
But, if you pass some additional args it will be a custom entry point or even entry points (if you wanna make few your blogs, for example. idk, is it relevant case?):
```
$ underblog ~/Docs/myBlog
```
will expect templates and MD it the `~/Docs/myBlog`, or

```
$ underblog ./myCatBlogRU ./myCatBlogEN 
```
just will build two blogs.

or we may just bail on the this feature :D :D

**Comment by freetonik** [comment]
I agree with the complete entry point as param.
